datafusion/core/src/physical_optimizer/sort_enforcement.rs

```diff
@@ -801,6 +803,22 @@ fn get_sort_exprs(sort_any: &Arc<dyn ExecutionPlan>) -> Result<&[PhysicalSortExp
     }
 }
 
+/// gets
```

Comment:
I think you forgot to complete this comment

Reply:
thanks